src/python/create.py

```diff
@@ -159,10 +160,47 @@ def create_materials(parsed_materials, project, data, citations):
                 material_dict["identifiers"].append(identifier)
 
             elif cell_type == "property":
+                if parsed_cell["key"] == "use_existing":
+                    use_existing = (
+                        True
+                        if parsed_cell["value"] in [True, "TRUE", "true", "True"]
```

Comment:
This is something you might reuse and could go in a function.

Reply:
okay, I can make it a function.

Reply:
also since python bools default to first char capital case you can use string.title() and just check on `True` and `"TRUE"`